src/main/java/net/pms/dlna/DLNAMediaInfo.java

```diff
@@ -86,6 +88,29 @@
 	 */
 	public static final long TRANS_SIZE = Long.MAX_VALUE - Integer.MAX_VALUE - 1;
 
+	/**
+	 * Containers that can represent audio or video media is by default
+	 * considered to be video. This {@link Map} maps such containers to the type
+	 * to use if they represent audio media.
+	 */
+	protected static final Map<String, String> audioOrVideoContainers;
+
+	static {
+		Map<String, String> mutableAudioOrVideoContainers = new HashMap<String, String>();
+
+		// Map container formats to their "audio variant".
+		mutableAudioOrVideoContainers.put(FormatConfiguration.MP4, FormatConfiguration.M4A);
+		mutableAudioOrVideoContainers.put(FormatConfiguration.MKV, FormatConfiguration.MKA);
+		mutableAudioOrVideoContainers.put(FormatConfiguration.OGG, FormatConfiguration.OGA);
+		mutableAudioOrVideoContainers.put(FormatConfiguration.RM, FormatConfiguration.RA);
+		mutableAudioOrVideoContainers.put(FormatConfiguration.MPEG1, FormatConfiguration.MPA);
+		mutableAudioOrVideoContainers.put(FormatConfiguration.MPEG2, FormatConfiguration.MPA);
```

Comment:
Fixed, I think. I don't know if it's correctly parsed etc, I've just added it to the "dual container" handling.